### PR TITLE
[KOGITO-4459] Removing trailing slash from springboot "raw result" endpoints

### PR DESCRIPTION
**Pull request by gitgabrio**
@danielezonca @tkobayas  @jiripetrlik

See https://issues.redhat.com/browse/KOGITO-4459

Needed by https://github.com/kiegroup/kogito-examples/pull/580

### CODE REVIEW

File: kogito-codegen-modules/kogito-codegen-predictions/src/main/resources/class-templates/PMMLRestResourceQuarkusTemplate.java [hunk 1]
```diff
@@ -32,7 +32,7 @@
     Application application;
 
     @POST
-    @Path("/")
+    @Path("")
```
Comment (danielezonca) [inline kogito-codegen-modules/kogito-codegen-predictions/src/main/resources/class-templates/PMMLRestResourceQuarkusTemplate.java:35]: I think you can remove this annotation completely in this case

**Review by danielezonca — APPROVED** [review]
Thanks, approving with minor comment

**Review by tkobayas — APPROVED** [review]
Thanks!

File: kogito-codegen-modules/kogito-codegen-predictions/src/main/resources/class-templates/PMMLRestResourceQuarkusTemplate.java (hunk 1 above)
Comment (gitgabrio) [reply kogito-codegen-modules/kogito-codegen-predictions/src/main/resources/class-templates/PMMLRestResourceQuarkusTemplate.java:35]: @danielezonca no it is required

**Comment by gitgabrio** [comment]
Jenkins retest this

**Comment by gitgabrio** [comment]
Jenkins retest this

**Review by jiripetrlik — APPROVED** [review]

**Pull request merged**